src/main/java/uk/gov/hmcts/cmc/claimstore/repositories/ClaimRepository.java

```diff
@@ -154,6 +154,15 @@ Integer linkSealedClaimDocument(
         @Bind("documentSelfPath") String documentSelfPath
     );
 
+    @SqlUpdate(
+        "UPDATE claim SET claim_document_store = :claimDocumentStore::JSONB"
+            + " WHERE id = :claimId"
+    )
+    Integer linkClaimToDocument(
```

Comment:
this can be addDocumentsToClaim

Reply:
we link only one document at a time with the claim ..The reason I felt it was appropriate to use linkClaimToDocument was that the previous method was called linkSealedClaimDocument.